ceph-salt-formula/salt/ceph-salt/cephtools.sls

```diff
@@ -16,13 +16,11 @@ install cephadm:
 {{ macros.end_step('Install cephadm and other packages') }}
 
 {{ macros.begin_step('Download ceph container image') }}
-{% if 'container' in pillar['ceph-salt'] and 'ceph' in pillar['ceph-salt']['container']['images'] %}
```

Comment:
Now that we guarantee `ceph-salt` will always pull the image beforehand,  we should call `cephadm bootstrap` command with the `--skip-pull`  option.

Reply:
Done